src/main.ts

```diff
@@ -197,11 +199,6 @@ const createWindow = async () => {
     },
   })
 
-  handlers.initialize({
-    browserWindow: mainWindow,
-    browserServer,
-  })
```

Comment:
@allansson I moved this outside of `createWindow` as calling it inside re-attempts to register the IPC handlers when the window is restored from the dock.

```
11:47:03.599 › Unhandled rejection Error: Attempted to register a second handler for 'auth:get-profiles'
    at IpcMainImpl.handle (node:electron/js2c/browser_init:2:100357)
```

I also removed `browserWindow` as a dependency since we're not using it right now. Let me know if this disrupts any work you're doing.

Reply:
The `browserWindow` is a relic from a previous iteration before there was a `browserServer` so removing it is the right call.